File: program2.c
#include <stdio.h>

int main() {
    int n, i;
    long a = 0, b = 1, c;

    printf("Enter number of terms: ");
    scanf("%d", &n);

    printf("Fibonacci Series: %ld %ld ", a, b);

    for (i = 3; i <= n; i++) {
        c = a + b;
        printf("%ld ", c);
        a = b;
        b = c;
    }

    printf("\n");
    return 0;
}

File: program2.py
# program2.py
# Fibonacci series without recursion

n = int(input("Enter how many Fibonacci numbers you want: "))

a, b = 0, 1

if n <= 0:
    print("Enter a positive number.")
elif n == 1:
    print("Fibonacci series:", a)
else:
    print("Fibonacci series:", end=" ")
    print(a, b, end=" ")

    for _ in range(2, n):
        c = a + b
        print(c, end=" ")
        a = b
        b = c
